website_menu_by_user_status/models/website_menu.py

```diff
@@ -14,27 +14,28 @@ class WebsiteMenu(models.Model):
     user_logged = fields.Boolean(
         string="User Logged",
         default=True,
-        help=_("If checked, "
-               "the menu will be displayed when the user is logged "
-               "and give access.")
+        help="If checked, "
+        "the menu will be displayed when the user is logged "
+        "and give access.",
     )
 
     user_not_logged = fields.Boolean(
         string="User Not Logged",
         default=True,
-        help=_("If checked, "
-               "the menu will be displayed when the user is not logged "
-               "and give access.")
+        help="If checked, "
+        "the menu will be displayed when the user is not logged "
```

Comment:
```suggestion
             "the menu will be displayed when the user is not logged "
```

Reply:
pep8 is complaining if I indent this.

Reply:
Our linters shouldn't.

Reply:
website_menu_by_user_status/models/website_menu.py:18:13: E131 continuation line unaligned for hanging indent
website_menu_by_user_status/models/website_menu.py:26:13: E131 continuation line unaligned for hanging indent